pkg/mount/block/blockdev_linux.go

```diff
@@ -604,27 +654,27 @@ func (b BlockDevices) FilterBlockPCI(blocklist pci.Devices) BlockDevices {
 	return partitions
 }
 
-// parsePCIBlockList parses a string in the format vendor:device,vendor:device
-// and returns a list of PCI devices containing the vendor and device pairs to block.
-func parsePCIBlockList(blockList string) (pci.Devices, error) {
+// parsePCIList parses a string in the format vendor:device,vendor:device
+// and returns a list of PCI devices containing the vendor and device pairs.
+func parsePCIList(parseList string) (pci.Devices, error) {
 	pciList := pci.Devices{}
-	bL := strings.Split(blockList, ",")
+	bL := strings.Split(parseList, ",")
 	for _, b := range bL {
 		p := strings.Split(b, ":")
 		if len(p) != 2 {
-			return nil, fmt.Errorf("BlockList needs to be of format vendor1:device1,vendor2:device2...! got %v", blockList)
+			return nil, fmt.Errorf("%w: %q", ErrListFormat, parseList)
 		}
 		// Check that values are hex and convert them to sysfs formats
 		// This accepts 0xABCD and turns it into 0xabcd
 		// abcd also turns into 0xabcd
 		v, err := strconv.ParseUint(strings.TrimPrefix(p[0], "0x"), 16, 16)
 		if err != nil {
-			return nil, fmt.Errorf("BlockList needs to contain a hex vendor ID, got %v, err %v", p[0], err)
+			return nil, fmt.Errorf("%w, got %v, err %v", ErrNotHex, p[0], err)
```

Comment:
sorry, now that I look at this again, I realize you don't need the ErrNotHex at all:
https://go.dev/play/p/pjGzJuwIz4O
also, the convention, which I think is usually worth following, would make the format like this:
fmt.Errorf("parsing blocklist %q:%w", p[0], err)
i.e. in general what it was doing, and what the error was. I like %q in places like this b/c there may be some weird characters in the string.
the strconv error is ErrSyntax, which you can check against. 

Again, the problem with returning an error saying "it has to be hex" is confusing:
v, err := strconv.ParseUint("abcdef000000000000", 16, 16)
https://go.dev/play/p/zGAochvjmlx
it's hex, so telling them it has to be hex is not right. strconv will return
value out of range
So I'd argue you don't want to tell them it has to be hex, b/c it may have BEEN hex.

Reply:
Done.